### PR TITLE
Provide the ability to run benchmarks with or without IPEX

### PR DESCRIPTION
**Pull request by anmyachev**
This PR is needed for https://github.com/intel/intel-xpu-backend-for-triton/pull/2200.

Closes https://github.com/intel/intel-xpu-backend-for-triton/issues/2204

### CODE REVIEW

**Comment by vlad-penkin** [comment]
@anmyachev please trigger to Benchmark CI runs for both paths in your feature branch and report run results back.

File: .github/workflows/triton-benchmarks.yml [hunk 1]
```diff
@@ -49,10 +53,20 @@ jobs:
         run: |
           pip install wheel
 
-      - name: Setup PyTorch
+      - name: Setup PyTorch with IPEX
+        if: ${{ inputs.install_ipex }}
```
[the remark this answers is not in the document]
Comment (anmyachev) [reply .github/workflows/triton-benchmarks.yml:57]: thanks @pbchekin!

Comment (anmyachev) [inline .github/workflows/triton-benchmarks.yml:57]:
```suggestion
        if: ${{ github.event_name == 'schedule' || inputs.install_ipex }}
```

File: .github/workflows/triton-benchmarks.yml [hunk 2]
```diff
@@ -49,10 +53,20 @@ jobs:
...
         uses: ./.github/actions/setup-pytorch
+        with:
+          repository: Stonepia/pytorch
+
+      - name: Setup PyTorch without IPEX
+        if: ${{ !inputs.install_ipex }}
```
Comment (anmyachev) [inline .github/workflows/triton-benchmarks.yml:63]:
```suggestion
        if: ${{ !(github.event_name == 'schedule' || inputs.install_ipex) }}
```

File: .github/workflows/triton-benchmarks.yml [hunk 3]
```diff
@@ -49,10 +53,20 @@ jobs:
...
+        uses: ./.github/actions/setup-pytorch
+        with:
+          repository: pytorch/pytorch
 
       - name: Setup IPEX
+        if: ${{ inputs.install_ipex }}
```
Comment (anmyachev) [inline .github/workflows/triton-benchmarks.yml:69]:
```suggestion
        if: ${{ github.event_name == 'schedule' || inputs.install_ipex }}
```

**Review by pbchekin — APPROVED** [review]

**Review by vlad-penkin — APPROVED** [review]

**Pull request merged**